graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/nodetype/GqlJcrNodeType.java

```diff
@@ -184,10 +185,18 @@ public List<GqlJcrNodeType> getSupertypes(@GraphQLName("fieldFilter") @GraphQLDe
     }
 
     @GraphQLField
-    @GraphQLName("mixinExtendsNames")
-    @GraphQLDescription("Returns the names of the mixin extends of this node type.")
-    public List<String> mixinExtendsNames() {
-        return nodeType.getMixinExtendNames();
+    @GraphQLName("mixinExtends")
+    @GraphQLDescription("Returns the mixin node types extended by this node type, filtered by the specified parameters.")
```

Comment:
Not sure that the wording is appropriate. It's actually returning the mixin node types extending this node type ?

Reply:
hmm, maybe I'm not understanding this _extends_ concept then...
With [this definition](https://github.com/Jahia/jahia/blob/6504c3ad858a47bab8d5fe3be9c043287bd11f30/test/core-test-framework/src/main/resources/WEB-INF/etc/repository/nodetypes/98-default-definitions.cnd#L257):
```
[jmix:internalLink] > jmix:templateMixin mixin
 extends = jnt:content
```
It means that `jmix:internalLink` extends `jnt:content`, right? In other words, that `jnt:content` is extended by the node type `jmix:internalLink`, no? 🤔